packages/atomic/src/components/recommendations/atomic-recs-list/atomic-recs-list.tsx

```diff
@@ -56,7 +63,8 @@ export class AtomicRecsList implements InitializableComponent<RecsBindings> {
   private nextNewResultTarget!: FocusTargetController;
 
   /**
-   * The desired layout to use when displaying results. Layouts affect how many results to display per row and how visually distinct they are from each other.
+   * How children results should be displayed. This does not affect the display of the list itself.
+   * To change the number of recommendations per column, change the -atomic-recs-number-of-columns CSS variable.
```

Comment:
Am I wrong or are there two dashes in the prefix of the CSS varialbles?
```suggestion
   * To modify the number of recommendations per column, modify the --atomic-recs-number-of-columns CSS variable.
```